code/game/objects/items/tanks/watertank.dm

```diff
@@ -171,6 +171,37 @@
 /obj/item/reagent_containers/spray/mister/janitor/mode_change_message(mob/user)
 	to_chat(user, span_notice("You [amount_per_transfer_from_this == 10 ? "remove" : "fix"] the nozzle. You'll now use [amount_per_transfer_from_this] units per spray."))
 
+//Security tank
+/obj/item/watertank/security
```

Comment:
Maybe rename this into /obj/item/watertank/pepperspray so to open the way for other security-oriented backpack watertanks?